torch_xla/csrc/ops/std_mean.cpp

```diff
@@ -0,0 +1,76 @@
+#include "torch_xla/csrc/ops/std_mean.h"
+
+#include "absl/strings/str_join.h"
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/reduction.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::XlaOp LowerStd(xla::XlaOp input,
+                    const std::vector<xla::int64>& dimensions,
+                    bool keep_reduced_dimensions,
+                    xla::int64 correction) {
+  return BuildStdDeviation(input, dimensions, keep_reduced_dimensions, correction);
+}
+
+xla::XlaOp LowerMean(xla::XlaOp input,
+                     const std::vector<xla::int64>& dimensions,
+                     bool keep_reduced_dimensions) {
+  return BuildMean(input, dimensions, keep_reduced_dimensions);
+}
```

Comment:
You don't really need these two helper function if they are just forwarding argument to `BuildStdDeviation` and `BuildMean`. You only replace the use of these two function with `Buildxxx` directly.

Reply:
addressed